flexqos.sh

```diff
@@ -801,7 +801,7 @@ prompt_restart() {
 } # prompt_restart
 
 menu() {
-	# clear
+	clear
```

Comment:
not sure if you want it to 'clear', but if not, please merge and comment clear later.